src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorArgumentDefinitions.java

```diff
@@ -48,6 +48,11 @@
      */
     public static final String ANNOTATION_OVERRIDES_LONG_NAME = "annotation-override";
 
+    /**
+     * List of final rendered fields to exclude from a final rendering
+     */
+    public static final String EXCLUSION_FIELDS_LONG_NAME = "exclusion-list";
```

Comment:
I'm not sure `exclusion-list` is a clear name.  Maybe something like "--skip-field" would be clearer.

Reply:
I like `--exclude-field`

Done